src/js/components/ServicePlanProgressBar.js

```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import ServicePlan from '../structs/ServicePlan';
+
+const labelMap = {
+  'InProgress': 'Updating'
+};
+
+class ServicePlanProgressBar extends React.Component {
+  getPhaseLabel(servicePlan) {
+    let phases = servicePlan.getPhases();
+
+    return (
+      <span className="text-overflow-break-word">
+        Phase {phases.getActiveIndex()} of {phases.getItems().length}:
```

Comment:
I think it would be fine to do:
```js
{`Phase ${phases.getActiveIndex()} of ${phases.getItems().length}: ${phases.getActive().getName()} `}
```
Or compute it above the view definition